Amélioration de l'UI de notre design system (storybook)

RomainSav opened the pull request.

RomainSav committed:
🎨 style(.storybook): add meta tags and styles to improve design system documentation
✨ feat(.storybook): add custom theme to Storybook manager UI
📦 chore(package.json): add @storybook/theming package as a dependency
The meta tags and styles added to the manager-head.html file improve the design system documentation by providing a more complete and consistent experience for users. The custom theme added to the Storybook manager UI improves the branding and visual consistency of the design system. The @storybook/theming package is added as a dependency to enable the use of custom themes in Storybook.

Bluzzi commented on .storybook/theme.ts:
```diff
@@ -0,0 +1,29 @@
+  appBorderColor: "#1E1E1F",
+  appBorderRadius: 4,
+
+  // Toolbar colors :
```
En anglais les `:` sont collés ! :p

Bluzzi commented on .storybook/manager-head.html:
```diff
@@ -0,0 +1,29 @@
+<meta name="description" content="Le système de conception du Royaume offre une expérience utilisateur cohérente et esthétique à travers notre site internet, grâce à des principes, composants et directives de conception réutilisables.">
+
+<meta name="og:title" content="Système de conception du Royaume">
+<meta name="og:description" content="Le système de conception du Royaume offre une expérience utilisateur cohérente et esthétique à travers notre site internet, grâce à des principes, composants et directives de conception réutilisables.">
```
Je pense qu'il est mieux d'utiliser le therme "Design System" même en français !

Bluzzi reviewed (approved):
Tu peux merge quand tu à check les deux points de la review.

RomainSav committed:
📝 docs(storybook): update meta tags and theme configuration
The meta tags in the `.storybook/manager-head.html` file have been updated to reflect the correct name of the design system, which is "Design system du Royaume" in French. The `theme.ts` file has been updated to improve the readability of the comments by removing unnecessary spaces and adding a colon after the section titles. No functional changes have been made.

RomainSav committed:
🔥 chore(.gitignore): remove storybook-static from gitignore
🚚 chore(.storybook/theme.ts): import create from @storybook/theming instead of @storybook/theming/create
The storybook-static directory is no longer being ignored as it is now required to be tracked in the repository. The import statement in the theme.ts file has been updated to import the create function from the @storybook/theming package instead of @storybook/theming/create. This is because the create function has been moved to the root of the package.
